app/validators/partner_details/answers_validator.rb

```diff
@@ -3,7 +3,9 @@ class AnswersValidator < BaseAnswerValidator
     include TypeOfMeansAssessment
 
     def applicable?
-      crime_application.client_has_partner == 'yes'
+      return false if applicant&.under18? || not_means_tested? || appeal_no_changes?
+
+      crime_application.client_has_partner != 'no'
```

Comment:
Unsure on this one, but thought if it is `nil`, the section is still applicable, just we don't yet know if client has a partner yet